steward/src/gas.rs

```diff
@@ -52,4 +76,13 @@ impl CellarGas {
         let data = etherchain_oracle.fetch().await;
         data
     }
+
+    async fn get_etherscan_client() -> Result<Client, Error> {
+        let config = APP.config();
+        let url = &config.ethereum.rpc;
+        let provider = crate::utils::get_eth_provider(url.as_str()).await?;
+        let chain = utils::get_chain(provider.clone()).await?;
```

Comment:
Nit: I think it's a bit more confusing for this and the above line to have fully specified crate paths rather than just importing the functions themselves, especially with two different "utils" packages.